### ISSUE TITLE
Add parallelized histogramming of pixel intensities

### ISSUE THREAD

**Issue by mattmeehan** [WIPACrepo/decotools#49]
We often histogram pixel intensities when analyzing DECO data. While this is a simple task on its own, it can be time-consuming to histogram many images in serial. I think it would be useful to have a function that parallelizes this using dask. The function would just take a list of image files as input and return the corresponding histograms, e.g.:
```python
def get_intensity_histograms(files, rgb_sum=False, cumulative=False, n_jobs=1):
    ...
```
We'd have parameters to determine the rgb conversion, cumulative vs differential distribution, and number of processes for parallelization. 

@jrbourbeau What do you think? And any other parameters or features that would be useful?

**Comment by jrbourbeau** [comment]
This sounds like a good addition to me! Organizationally, it might be worth adding a new module, perhaps named `decotools.metrics`, and putting the current `blob_extraction.get_intensity_metrics` and the new `get_intensity_histograms` in there.

**Comment by mattmeehan** [comment]
Great. Yeah, I think that's a good way to organize it too.

**Comment by jrbourbeau** [comment]
Fixed in #53
